.golangci.yml

```diff
@@ -64,8 +64,6 @@ linters:
     - forbidigo
     - forcetypeassert
     - gci
-    - gochecknoglobals
```

Comment:
I looked at the linter docs and found no way to have a config to allow certain global variables, and it was the same for other linter as well.